### ISSUE TITLE
cross compiler libvncserver for arm-linux

### ISSUE THREAD

**Issue by wadexu** [LibVNC/libvncserver#62]
May I know the commands to cross compiler the libvncserver on ubuntu, my target machine is omap5 (arm core).

**Comment by maxnet** [comment]
Easiest way might be to use buildroot, which has a libvncserver package.

**Comment by wadexu** [comment]
Do you means build the buildroot for arm? The libvncserver is part of buildroot?

Is there a built binary for arm available to download?

**Comment by maxnet** [comment]
http://www.buildroot.org/
-> make menuconfig
-> select your CPU/target and toolchain, and choose libvncserver under "package selection"
-> make

**Comment by wadexu** [comment]
Thank you. I used the buildroot to compiled the libvncserver successfully.
However, I can't find the exec file named "linuxvnc". 
I browsed the git at "https://github.com/LibVNC/libvncserver/tree/LibVNCServer-0.9.10", it seems have 2 branch. One is "LibVNCServer_X.X.X" another one is "X11VNC_X_X_X", the X11VNC have a directory call "vncterm", is this directory the source code for "linuxvnc" application? Seems I can't find it in Buildroot packages
